qa/L0_infer/test.sh

```diff
@@ -331,6 +342,102 @@ for TARGET in cpu gpu; do
     set -e
 done
 
+# Run 'python', 'python_dlpack' and 'onnx' models separately in valgrind test.
+# Loading python and python_dlpack models has OOM issue when running with
+# valgrind, so loading only batch or nobatch models for each time.
+# Loading all the onnx models at once requires more than 12 hours. Loading them
+# separately to reduce the loading time.
+if [ "$TEST_VALGRIND" -eq 1 ]; then
+  TESTING_BACKENDS="python python_dlpack onnx"
+  if [ "$TEST_JETSON" == "0" ]; then
+    if [[ "aarch64" != $(uname -m) ]] ; then
+        pip3 install torch==1.13.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
+    else
+        pip3 install torch==1.13.0 -f https://download.pytorch.org/whl/torch_stable.html
+    fi
+  fi
+
+  for BACKENDS in $TESTING_BACKENDS; do
```

Comment:
BACKENDS -> FW 

For consistency.

Reply:
The `BACKENDS` env is exported and used in [infer_test.py](https://github.com/triton-inference-server/server/pull/5502/files#diff-59676af454541b48dd8e7c83879c5f50c8994c0e3161d9801f05b23b9b7e3e94L50). It's set to all backends [here](https://github.com/triton-inference-server/server/pull/5502/files#diff-66b8024d0503d76eac74493835a3d7a6e192903c3daca39f00c54fa96e9120b9R130) in the `L0_infer--base` test. I think it would make sense to use `BACKENDS`  so that we can export the env directly. Let me know your thought on this.